Gemfile

```diff
@@ -25,10 +25,10 @@ end
 
 group :development, :test do
   gem "byebug", "~> 10"
-  gem "capybara", "~> 3"
   gem "factory_bot_rails", "~> 4"
   gem "govuk-lint", "~> 3"
   gem "govuk_schemas", "~> 3.2"
+  gem "govuk_test", "~> 0.1"
```

Comment:
Can this be split out into a separate PR? (It seems to be independent of this one.)

Reply:
We added it so that we could make use of the selenium-webdriver gem in order to test using Javascript. @alex-ju is looking into getting the Capybara tests to play nicely with Javascript as having it enabled is breaking them at the moment.